packages/blade/src/tokens/theme/paymentTheme.ts

```diff
@@ -1833,19 +1781,19 @@ const colors: ColorsWithModes = {
             primary: {
               default: {
                 lowContrast: globalColors.chromatic.sapphire[500],
-                highContrast: globalColors.chromatic.sapphire[600],
+                highContrast: globalColors.chromatic.sapphire[50],
```

Comment:
why this change? this is being done by plugin already right?

Reply:
Plugin has only done this not me 🤔